addons/stock/static/src/views/stock_empty_list_help.js

```diff
@@ -0,0 +1,10 @@
+import { registry } from "@web/core/registry";
+import { useActionLinks } from "@web/views/view_hook";
+
+export class StockEmptyList {
```

Comment:
a view is not a class but an object that describes the view.

grep `registry\.category\(["']views` in regex mode to see how it is done.
You would have to extend
- make StockEmptyList (rename it to something like: InteractiveActionHelp) a owlComponent
- extend the Renderer, extend its component object, push  InteractiveActionHelp in that object.
-

Reply:
Thank you for your review @kebeclibre ! I've amended the code, using `LoyaltyActionHelper` as an example. It makes a bit more sense now, I hope :sweat_smile: 

However, I am still getting an error, when I try to open an empty list:

```
UncaughtPromiseError > OwlError
Uncaught Promise > An error occured in the owl lifecycle (see this Error's "cause" property)
OwlError: An error occured in the owl lifecycle (see this Error's "cause" property)
    at handleError (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:947:101)
    at App.handleError (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1573:29)
    at Fiber._render (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:972:19)
    at Fiber.render (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:970:6)
    at ComponentNode.initiateRender (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1038:47)

Caused by: TypeError: Cannot destructure property 'resModel' of 'undefined' as it is undefined.
    at useActionLinks (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:10178:66)
    at StockActionHelper.setup (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:15923:655)
    at new ComponentNode (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1033:202)
    at http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1571:6
    at StockListRenderer.template (eval at compile (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1531:374), <anonymous>:104:12)
    at node.renderFn (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1088:210)
    at Fiber._render (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:971:96)
    at Fiber.render (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:970:6)
    at ComponentNode.initiateRender (http://localhost:8069/web/assets/bb33150/web.assets_web.min.js:1038:47)
    
```

Did I miss a part of the configuration?

Reply:
@kebeclibre I've added a model (similar as in other examples in our codebase) and the error is gone :tada: However, the empty list help message is not showing correctly anymore :cry: I assume that I've messed something up with new component's properties, I'll investigate further tomorrow.